autorevision.sh

```diff
@@ -1029,7 +1029,7 @@ pathSegment() {
 	local depth="0"
 
 	if [ ! -z "${pathz}" ]; then
-		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ]; do
+		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ] && [ ! "${pathz}" = "C:" ]; do
```

Comment:
What happen if instead of `[ ! "${pathz}" = "C:" ]` you have `[ ! "${pathz}" = "" ]`?